### DISCUSSION TITLE
Bazel vs. Tuist for large scale iOS apps by small-medium size teams

### DISCUSSION THREAD

**Discussion by shahzadmajeed** [MobileNativeFoundation/discussions#170] [General]
I'm curious how do you modularize your apps, manage your projects and which build system do you use. Just to define some format:

Company: 
Team Size:
Modularization detail:
App architecture(MVVM/VIPER/RIBS/TCA etc.):
Build system:
Any additional tools or details:

These data points will help us making a decision for long term large scale projects.

**Comment by shahzadmajeed** [comment]
I have done some R&D on Tuist and found it to be pretty good. I also read some articles on Bazel (but didn't try it out yet) which make me think that Bazel is a complex setup for iOS projects and suitable for big teams. Am I right to conclude this?

Why would you prefer Bazel over Tuist?

**Comment by shahzadmajeed** [comment]
@keith I would like to get some insights/feedback/advice on this topic from MNF member companies as they seem to be using Bazel. How do I get this community/foundation's attention please?

**Reply by keith** [reply]
Hey! Yea there's a lot to talk about here, it's really up to individuals to jump in and provide context for their companies. For these topics specifically there are a few mentions of this in previous conversations such as https://github.com/MobileNativeFoundation/discussions/discussions/31 and https://github.com/MobileNativeFoundation/discussions/discussions/42
